firmware/lib/Comm/src/TestStandCommController.h

```diff
@@ -17,12 +19,11 @@ class TestStandCommController : public TestStandComm
     public:
         TestStandCommController(SerialDevice& device);
 
-        SerialResult ping();
         SerialResult log(LogLevel log_level, const char *fmt, ...);
         SerialResult status(Status status);
         SerialResult position(int32_t x_counts, int32_t y_counts);
         SerialResult axis_state(/*TODO*/);
```

Comment:
Will there be another PR with this implementation?

Reply:
Yup